src/main/java/org/jabref/logic/exporter/MSBibExporter.java

```diff
@@ -52,7 +52,7 @@ public void performExport(final BibDatabaseContext databaseContext, final String
             } catch (TransformerException | IllegalArgumentException | TransformerFactoryConfigurationError e) {
                 throw new Error(e);
```

Comment:
Not your code, I know, but isn't it weird to throw an error here? Wouldn't it be better at least scale that down to a `RuntimeException`?